addon/templates/components/frost-object-details.hbs

```diff
@@ -1,23 +1,33 @@
-{{yield}}
-<div class="view-tabs">
-    <div class="tabs">
-      {{#yield-slot 'view-route'}}
-        {{yield (block-params (hash
-        view=(component 'frost-view-detail')))}}
-      {{/yield-slot}}
+<div data-test={{hook (concat hook '-object-details')}}>
+  <div data-test={{hook (concat hook '-object-details-content')}}>
+    {{yield}}
+  </div>
+  <div class="object-details-tabs" data-test={{hook (concat hook '-object-details-tabs')}}>
+    {{#each detailTabs as |tab|}}
+      {{component tab
+        hook=hook
```

Comment:
shouldn't this hook be customised with an index of the tab? Something like `(concat hook '-object-details-tabs' index)`